pkg/api/handlers/libpod/images.go

```diff
@@ -270,6 +270,16 @@ func ExportImages(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	// if format is dir, server will save to an archive
+	// the client will unArchive after receive the archive file
+	// so must convert is at here
+	if query.Format == define.OCIManifestDir {
+		query.Format = define.OCIArchive
+	}
+	if query.Format == define.V2s2ManifestDir {
+		query.Format = define.V2s2Archive
+	}
```

Comment:
Non-blocking: a `switch query.Format {...}` may be more idiomatic.

LGTM

Reply:
had done